gradle/docs.gradle

```diff
@@ -9,13 +9,13 @@ tasks.withType(Javadoc).configureEach {
     }
 }
 
-tasks.register('javadocJar', Jar) {
-    dependsOn javadoc
+def javadocJar = tasks.register('javadocJar', Jar) {
+    dependsOn tasks.withType(Javadoc)
     archiveClassifier.set('javadoc')
     from 'build/docs/javadoc'
 }
 
-tasks.register('sourcesJar', Jar) {
+def sourcesJar = tasks.register('sourcesJar', Jar) {
```

Comment:
I think we can drop these tasks entirely and instead use
```
java {
    withJavadocJar()
    withSourcesJar()
}
```

Reply:
Good catch!